content_api/publish/service.py

```diff
@@ -30,10 +30,22 @@ def create(self, docs, **kwargs):
             doc[config.ID_FIELD] = doc['guid']
             del doc['guid']
             _id = doc[config.ID_FIELD]
-            original = self.find_one(req=None, _id=_id)
-            if original:
-                self.update(_id, doc, original)
-                ids.append(_id)
-            else:
-                ids.extend(super().create([doc], **kwargs))
+            ids.extend(self._create_doc(doc, **kwargs))
         return ids
+
+    def _create_doc(self, doc, **kwargs):
+        _id = doc[config.ID_FIELD]
+        original = self.find_one(req=None, _id=_id)
+        self._process_associations(doc)
+        if original:
+            self.update(_id, doc, original)
+            return _id
+        else:
+            return super().create([doc], **kwargs)
+
+    def _process_associations(self, doc):
+        if 'associations' in doc:
+            for assoc_list in doc['associations'].values():
+                for assoc in assoc_list:
+                    if len(assoc) > 2:
```

Comment:
Association dict contains "_id" and "type" fields. When the item was embedded it contains more than 2 fields.